tests/models/abiotic/test_abiotic_tools.py

```diff
@@ -0,0 +1,54 @@
+"""Test abiotic_tools.py."""
+
```

Comment:
The tests are cut and paste from the `wind` submodule, so they already went through PR process.